var/spack/repos/builtin/packages/llvm-amdgpu/package.py

```diff
@@ -94,12 +94,26 @@ def cmake_args(self):
             'compiler-rt'
         ]
 
+        if self.spec.satisfies('@4.2.0:'):
+            llvm_projects.append('libcxx')
+            llvm_projects.append('libcxxabi')
+
+            args = [
+                self.define('LIBCXX_ENABLE_SHARED', 'OFF'),
+                self.define('LIBCXX_ENABLE_STATIC', 'ON'),
+                self.define('LIBCXX_INSTALL_LIBRARY', 'OFF'),
+                self.define('LIBCXX_INSTALL_HEADERS', 'OFF'),
+                self.define('LIBCXXABI_ENABLE_SHARED', 'OFF'),
+                self.define('LIBCXXABI_ENABLE_STATIC', 'ON'),
+                self.define('LIBCXXABI_INSTALL_STATIC_LIBRARY', 'OFF'),
+                self.define('LLVM_ENABLE_ZLIB', 'ON'),
+                self.define('LLVM_ENABLE_Z3_SOLVER', 'OFF')
```

Comment:
why?

Reply:
These are default options which are used with the 4.3 release build. As per the comment from Sreeni, i have added the sam e here.

Reply:
Hm, it seems rather unrelated... z3 is used for the static analyzer and zlib is on by default, can you just remove those lines?